Minor fixes to AML environment customization docs (#119673)
Azure/azure-content: articles/machine-learning/prompt-flow/how-to-customize-environment-runtime.md

```
###### Requirements with Version Specifiers ######
langchain == 0.0.149        # Version Matching. Must be version 0.0.149
keyring >= 4.1.1            # Minimum version 4.1.1
coverage != 3.5             # Version Exclusion. Anything except version 3.5
Mopidy-Dirble ~= 1.1        # Compatible release. Same as >= 1.1, == 1.*
<path_to_local_package>     # reference to local pip wheel package
```

> See [Requirements File Format](https://pip.pypa.io/en/stable/reference/requirements-file-format/) in the pip documentation for more details on structuring the `requirements.txt` file.

Reviewer: ```suggestion
For more information about structuring the `requirements.txt` file, see [Requirements file format](https://pip.pypa.io/en/stable/reference/requirements-file-format/) in the pip documentation.
```
Minor edits for Microsoft style
